docs/glossary.md

```diff
@@ -0,0 +1,75 @@
+## Glossary
```

Comment:
glossary could be sub-sectioned by:

- stuff about pipelines

- stuff about tiles, fovs, rounds, channels

- stuff about features, barcodes, codebooks, decoding

order probably matters to bulid a logical narrative flow.

Reply:
Do you think stuff about pipelines belongs here? or just a minimum to describe the objects of the spec, and more detail goes in a starfish glossary?

Reply:
i think so. then some pipeline spec doc and/or starfish thing can reference this one global glossary. we can explicitly link to sub-sections of it as well maybe.